Automattic/jetpack
# Comments: Adding and verifying a nonce field

#### Changes proposed in this Pull Request:

* This PR both creates and verifies the existence of a nonce for the Jetpack comments form.
* On the WordPress.com side in a separate diff, a hidden input field is created and the field is passed back to the site: D76657-code.

#### Jetpack product discussion

* This was [suggested in this PR comment](https://github.com/Automattic/jetpack/pull/22835#pullrequestreview-891447975).


#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* Though it's not a complete way to test, you should be able to see the `jetpack_comments_nonce` hidden input when viewing the HTML source for the comment's reply field.
* You also shouldn't see any 'nonce verification' failed errors when testing.
